### DISCUSSION TITLE
Reordering items in system tray

### DISCUSSION THREAD

**Discussion by foberle** [linuxmint/discussions#271] [Ideas]
It seems that almost everything in the panel can be configured except for the ordering of items in the System Tray. As an example of how this would be useful: placing the printer management next to the HP specific printer device manager (or similar) would help organize a "busy" panel. This could be limited to something like a gsettings entry, with a GUI addition to System Settings at some later time.

**Comment by AtomicRobotMan0101** [comment]
- Right click the panel --> Panel edit Mode
- Drag them around.

There are sometimes groups of icons that move as clumps. This seems to be controlled by XApp Status Applet (xapp-status@cinnamon.org) (anyone - verify please!).

This applets self-describes as "An applet which hosts all the system tray icons for applications which use XAppStatusIcons"

It is version 1.

I suspect this is something you are interested in.... (so am I).

......
Edit for some info Ive been reading:
https://projects.linuxmint.com/xapp/reference/XAppStatusIcon.html
